lib/Rex/Service/FreeBSD.pm

```diff
@@ -45,11 +45,13 @@ sub ensure {
   if ( $what =~ /^stop/ ) {
     $self->stop( $service, $options );
     delete_lines_matching "/etc/rc.conf",
-      matching => qr/${service}_enable="YES"/;
+      matching => qr/${service}_enable="?[Yy][Ee][Ss]"?/;
```

Comment:
Well, no. It doesn't. Just an old way (a bit shell-ish) to do it, sorry =).
> qr/${service}_enable="?YES"?/i (matches the whole expression case-insensitively)
We shouldn't make the whole expression case-insensitive. It doesn't make sense.
> qr/${service}_enable="?((?i)YES)"?/ (matches only the YES part case-insensitively)
is definitely looks better. I vote for this one! =)
Same story with line 54 below ;)